apps/explorer/lib/explorer/chain.ex

```diff
@@ -3738,19 +3738,29 @@ defmodule Explorer.Chain do
     |> Repo.all()
   end
 
-  def address_to_forward_transfers(address_hash, options \\ []) when is_list(options) do
```

Comment:
This method is used to display the forward transfers on the address page: https://github.com/search?q=repo%3AHorizenLabs%2Feon-explorer%20address_to_forward_transfers&type=code

It needs to be replaced w the new method

Reply:
good catch, thanks!